LocalChat/ChatViewController.swift

```diff
@@ -24,33 +27,98 @@ class ChatViewController: UIViewController, NumberOfSegmentRequiredDelegate {
     
     override func viewWillAppear(animated: Bool) {
         super.viewWillAppear(true)
+        
+        chatUserNameLabel?.text = String(numberOfSegment)
+        
+        self.frameView = UIView(frame: CGRectMake(0, 0, self.view.bounds.width, self.view.bounds.height))
+        
+        // Observer Methods for keyboard to show and hide
+        let center: NSNotificationCenter = NSNotificationCenter.defaultCenter()
+        center.addObserver(self, selector: #selector(keyboardWillShow(_:)), name: UIKeyboardWillShowNotification, object: nil)
```

Comment:
Remove( _:)
Follow : https://www.hackingwithswift.com/example-code/uikit/how-to-adjust-a-uiscrollview-to-fit-the-keyboard